### PR TITLE
Chore: Refactored game.ts and fix reaction

### LINKED ISSUE #478: Convert all commands related to games to the new event handler
This is currently blocked by #467

### PR DESCRIPTION
**Pull request by hasanlatch007**
I refactored what I felt was necessary for anything related to the old event handler. Not touching the hubs or other like just commands.
This will have a merge conflict with the PR #500 as the old reaction event handler had an issue picking up reaction added in DMs,
This should be fixed in this PR and also fixed in #500 as that file will be deleted

!closes #478

### CODE REVIEW

File: src/programs/game.ts [hunk 1]
```diff
@@ -1,43 +1,82 @@
...
-  await embedMessage.delete();
-
-  const emoji = selection.emoji.name;
-  try {
-    await hub.createSession(emoji, message.author.id);
-    await message.delete();
-  } catch (e) {
-    if (e instanceof Error) {
-      await Tools.handleUserError(message, e.message);
+}
```
Comment (adrian-goe) [inline src/programs/game.ts:30]: move into own file

File: src/programs/game.ts [hunk 2]
```diff
@@ -1,43 +1,82 @@
...
+      await hub.createSession(emoji, message.author.id);
+      await message.delete();
+    } catch (e) {
+      if (e instanceof Error) {
+        await Tools.handleUserError(message, e.message);
+      }
     }
   }
-};
+}
```
Comment (adrian-goe) [inline src/programs/game.ts:62]: move into own file

File: src/programs/game.ts [hunk 3]
```diff
@@ -1,43 +1,82 @@
...
+      hub.routeMessage(message);
+      return;
+    }
+
+    const entertainmentChannel = (message.channel as GuildChannel).parent.name
+      .toLocaleLowerCase()
+      .endsWith("entertainment");
+    if (entertainmentChannel) hub.routeMessage(message);
+  }
+}
```
Comment (adrian-goe) [inline src/programs/game.ts:82]: move into own file

**Review by adrian-goe — CHANGES_REQUESTED** [review]

**Comment by hasanlatch007** [comment]
Games have been removed from the bot. This PR can be closed

**Pull request closed without merging**